docs/notebooks/multiobjective_ehvi.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
```

Comment:
the notebooks should be in another PR, though I realise it's probably easier for you to leave them in this branch until we've finalised things a little more, so no need to move them yet.